test_app/models.py

```diff
@@ -47,3 +46,19 @@ class RelatedFieldsTestModel(CommonModel):
     more_teams = models.ManyToManyField(Team, related_name='related_fields_test_model_more_teams')
 
     ignore_relations = ['teams_with_no_view']
+
+
+class ImmutableLogEntry(ImmutableCommonModel):
+    """
+    Testing ImmutableModel with CommonModel
```

Comment:
They are and this led me to break out ImmutableCommonModel into an ImmutableModel (because we might want immutable models that don't inherit AbstractCommonModel, etc.)

So I've done that in the latest commit and I think this comment is back to making sense :rofl: